src/postgres/src/test/regress/expected/yb_pg_window.out

```diff
@@ -932,10 +1114,13 @@ FROM tenk1 WHERE unique1 < 10 WINDOW w AS (order by four) ORDER by 1,2,3;
   42 |       7 |    3
 (10 rows)
 
---SELECT first_value(unique1) over w,
---last_value(unique1) over w, unique1, four
---FROM tenk1 WHERE unique1 < 10
---WINDOW w AS (order by four range between current row and unbounded following)  ORDER by 1,2,3,4,5;
+SELECT first_value(unique1) over w,
+last_value(unique1) over w, unique1, four
+FROM tenk1 WHERE unique1 < 10
+WINDOW w AS (order by unique1 range between current row and unbounded following)  ORDER by 1,2,3,4,5;
+ERROR:  ORDER BY position 5 is not in select list
+LINE 4: ...en current row and unbounded following)  ORDER by 1,2,3,4,5;
```

Comment:
Is this an intentional test case where you expect provide a non-existent column position "5"?

Reply:
It isn't intentional, I have changed it and made sure the output matches what I have calculated in my latest commit.